### PR TITLE
Update MemoryMetricsDatastore import in docs

### PR DESCRIPTION
**Pull request by SotaNakajima**
Update documentation w.r.t. importing MemoryMetricsDatastore

### Category (place an `x` in each of the `[ ]`)

* [ ] **bolt** (Bolt for Java)
* [ ] **bolt-{sub modules}** (Bolt for Java - optional modules)
* [X] **slack-api-client** (Slack API Clients)
* [ ] **slack-api-model** (Slack API Data Models)
* [ ] **slack-api-*-kotlin-extension** (Kotlin Extensions for Slack API Clients)
* [ ] **slack-app-backend** (The primitive layer of Bolt for Java)

## Requirements

Please read the [Contributing guidelines](https://github.com/slackapi/java-slack-sdk/blob/main/.github/contributing.md) and [Code of Conduct](https://slackhq.github.io/code-of-conduct) before creating this issue or pull request. By submitting, you agree to those rules.

### CODE REVIEW

**Comment by mwbrooks** [comment]
Hi @SotaNakajima 👋🏻 Thanks a bunch for the contribution and welcome to the project 🎉 

Could you please [sign the Contributor License Agreement](https://github.com/slackapi/java-slack-sdk/pull/1083#issuecomment-1307825513) so that we can review and merge your pull request 🙇🏻

File: docs/guides/web-api-basics.md [hunk 1]
```diff
@@ -266,8 +266,7 @@ CompletableFuture<ChatPostMessageResponse> postMessageResult = slack.methodsAsyn
 If your app has multiple nodes, it's also possible to specify the number of nodes.
 
 ```java
-import com.slack.api.methods.metrics.MetricsDatastore;
-import com.slack.api.methods.metrics.impl.MemoryMetricsDatastore;
+import com.slack.api.methods.metrics.MemoryMetricsDatastore;
```
Comment (mwbrooks) [inline docs/guides/web-api-basics.md:269]: @WilliamBergamin or @seratch could you please confirm that it's safe to remove `MetricsDatastore` import?

Comment (WilliamBergamin) [reply docs/guides/web-api-basics.md:269]: @mwbrooks this looks good to me 🚀

**Review by WilliamBergamin — APPROVED** [review]

**Review by seratch — CHANGES_REQUESTED** [review]
@SotaNakajima Thanks for fixing this! Your changes look good to me but could you apply similar changes to the Redis-backend part? https://github.com/slackapi/java-slack-sdk/blame/c4f3906fb84057f44f876750e13ea62b8e7d2a9b/docs/guides/web-api-basics.md#L354 Also, signing with our CLA using your GitHub email address would be appreciated.

**Review by seratch — APPROVED** [review]
Thank you!

**Comment by SotaNakajima** [comment]
> Thank you!

こちらこそありがとう！

**Pull request merged**